src/main/kotlin/calculator/Params.kt

```diff
@@ -0,0 +1,34 @@
+package calculator
+
+class Params(text: String?) {
+    val paramList: List<Param>
+
+    init {
+        paramList = if (text.isNullOrBlank()) {
+            listOf(Param.EMPTY_PARAM)
+        } else {
+            val tokens = split(text)
+            tokens.map { Param(it) }
+        }
+    }
+
+    fun fold(operation: (Int, Int) -> Int): Param {
+        return paramList.fold(Param.EMPTY_PARAM) {
```

Comment:
```suggestion
        return paramList.reduce {
```
컬렉션의 첫 번째 값부터 fold할 때는 `reduce`를 사용할 수 있습니다 :)